samples/apps/copilot-chat-app/webapi/CopilotChatApi.csproj

```diff
@@ -13,6 +13,28 @@
     <UserSecretsId>aspnet-SKWebApi-1581687a-bee4-40ea-a886-ce22524aea88</UserSecretsId>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Remove="PlannerSkills\Notegen\config.json" />
```

Comment:
Remove, Include once, Include twice...  Something's fishy in this file

Reply:
+1